Automattic/wp-calypso
# Vdimitrakis/selfserve 222 replace promoted to blaze

#### Proposed Changes

This task is to rename "Promoted Posts" to "Blaze" and replacing some strings for the Calypso interface

proposed changes are documented below:

1. In pages list

before:
![Screenshot 2022-12-28 at 18 16 50](https://user-images.githubusercontent.com/1416426/210066531-c5b4f8d2-4309-4e6a-8a9a-1f4068d2ef2b.png)

after: 
![Screenshot 2022-12-28 at 18 16 22](https://user-images.githubusercontent.com/1416426/210066554-25295821-ce57-4665-ae29-816ff9a078f1.png)

2. in posts list

before:
![Screenshot 2022-12-28 at 18 15 42](https://user-images.githubusercontent.com/1416426/210066587-ce82e2d2-7d91-4c00-955f-f35787051a1e.png)

after: 
![Screenshot 2022-12-28 at 18 16 12](https://user-images.githubusercontent.com/1416426/210066593-b409a57c-e3d3-48ed-9e5d-2751516a4f67.png)

3. in widget header bar

before:
![Screenshot 2022-12-28 at 18 15 17](https://user-images.githubusercontent.com/1416426/210066616-8c97fa6c-4c10-4297-9ded-b6733e866e6f.png)

after: 
![Screenshot 2022-12-28 at 18 14 46](https://user-images.githubusercontent.com/1416426/210066630-d5cfa92c-f042-43ee-9bdc-0a11aef8f1e1.png)

4. in admin home 

before:
![Screenshot 2022-12-28 at 18 10 59](https://user-images.githubusercontent.com/1416426/210066673-a3037a59-2986-4120-966d-77fd45b9a38a.png)

after: 
![Screenshot 2022-12-28 at 18 11 30](https://user-images.githubusercontent.com/1416426/210066685-2b7f3ce8-d280-4331-a4d1-549e040cd638.png)

5. in admin home (features boxes)

before:
![Screenshot 2022-12-28 at 18 10 48](https://user-images.githubusercontent.com/1416426/210066739-26b0e51f-13b2-40df-b532-e2b982c6e777.png)

after: 
![Screenshot 2022-12-28 at 18 10 10](https://user-images.githubusercontent.com/1416426/210066757-89df4d78-cec1-4c61-9bc5-68d2e8e385da.png)

6. in /advertising/blog_url (in the list elements)

before:
![Screenshot 2022-12-28 at 18 09 34](https://user-images.githubusercontent.com/1416426/210066812-6f15a1f4-ac37-46a2-a18f-2be2d41fe5c9.png)

after: 
![Screenshot 2022-12-28 at 18 09 11](https://user-images.githubusercontent.com/1416426/210066837-93aea785-1d4b-4639-a4ab-45852dc5cc44.png)

changes highlighted:

![Screenshot 2022-12-28 at 18 09 11](https://user-images.githubusercontent.com/1416426/210069048-4dc712d6-d9db-4702-a69d-139d97605cef.png)

7. in /advertising/blog_url (when the site is private)

before: 
![Screenshot 2023-01-03 at 12 24 52](https://user-images.githubusercontent.com/1416426/210532319-ec393361-239e-48a3-853b-157fafcaa553.png)

after: 
![Screenshot 2023-01-04 at 12 08 01](https://user-images.githubusercontent.com/1416426/210532368-008974b1-9ee8-40b4-9168-7f5997af4bda.png)

8. in the same page (when the site is not published)

before: 
![Screenshot 2023-01-03 at 12 22 07](https://user-images.githubusercontent.com/1416426/210532484-42a226a7-dcf7-4a29-90a8-13ef6f441807.png)

after:
![Screenshot 2023-01-04 at 12 07 43](https://user-images.githubusercontent.com/1416426/210532513-568aadfc-c042-4489-b0c8-c7ffe557caea.png)


I am using the branch of the epic ([SELFSERVE-206) as the main branch to merge in trunk. and i am making other branches with changes so that we can review and approve those (and merge them in the epic main branch) to make it easier to review each change... when all is ok we can merge the main branch into trunk


#### Testing Instructions

I think most changes are straightforward and simple text replacements.. just checkout the branch and check that changes are ok and according to the figma files

#### Pre-merge Checklist





Related to #
